src/sentry/integrations/jira_server/integration.py

```diff
@@ -1286,7 +1356,13 @@ class JiraServerIntegrationProvider(IntegrationProvider):
 
     needs_default_identity = True
 
-    features = frozenset([IntegrationFeatures.ISSUE_BASIC, IntegrationFeatures.ISSUE_SYNC])
+    features = frozenset(
+        [
+            IntegrationFeatures.ISSUE_BASIC,
+            IntegrationFeatures.ISSUE_SYNC,
+            IntegrationFeatures.USER_MAPPING,
```

Comment:
This is here to act as a flag for the UI to render a tab view vs the single config pane we currently have. I'm open to suggestions on how to do this differently though.